### PR TITLE
removed unused files for db connection, updated hibernate.hbm2ddl.aut…

### PR DESCRIPTION
**Pull request by tburlakova**
…o to 'update'

### CODE REVIEW

File: src/main/java/net/javaguides/springmvc/controller/rest/UserController.java [hunk 1]
```diff
@@ -1,45 +1,57 @@
...
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping(value = "/users")
     @ResponseBody
     public User createUser(@RequestBody User user) {
         return userService.saveUser(user); }
 
+    @ResponseStatus(HttpStatus.OK)
     @GetMapping("/users/{id}")
-    public User findUserById(@PathVariable Integer id) throws UserNotFoundException {
+    public User findUserById(@PathVariable Integer id) throws ResourceNotFoundException {
```
[the remark this answers is not in the document]
Comment (tburlakova) [reply src/main/java/net/javaguides/springmvc/controller/rest/UserController.java]: done

File: src/main/java/net/javaguides/springmvc/controller/rest/UserController.java [hunk 2]
```diff
@@ -1,45 +1,57 @@
...
         return userService.getUser(id);
     }
 
+    @ResponseStatus(HttpStatus.OK)
     @PutMapping("/users/{id}")
-    public User updateUser(@PathVariable Integer id, @RequestBody User user) {
-        return userService.updateUser(id, user);
+    public User updateUser(@PathVariable Integer id, @RequestBody Set<Phone> phones) throws ResourceNotFoundException {
```
[the remark this answers is not in the document]
Comment (tburlakova) [reply src/main/java/net/javaguides/springmvc/controller/rest/UserController.java]: done

File: src/main/java/net/javaguides/springmvc/service/UserService.java [hunk 3]
```diff
@@ -1,18 +1,21 @@
...
+import java.util.Set;
 
 public interface UserService {
 
     Iterable<User> getUsers();
 
     User saveUser(User user);
 
-    User updateUser(int id, User user);
+    User updateUser(int id, Set<Phone> phones) throws ResourceNotFoundException;
```
[the remark this answers is not in the document]
Comment (tburlakova) [reply src/main/java/net/javaguides/springmvc/service/UserService.java]: done

File: src/main/java/net/javaguides/springmvc/controller/rest/UserController.java [hunk 4]
```diff
@@ -1,45 +1,64 @@
...
-    public User updateUser(@PathVariable Integer id, @RequestBody User user) {
-        return userService.updateUser(id, user);
+    @ResponseStatus(HttpStatus.CREATED)
+    @PostMapping("/users/{id}/phoneNumbers")
+    @ResponseBody
+    public User addPhone(@PathVariable Integer id, @RequestBody Set<Phone> phones) {
+        return userService.addPhone(id, phones);
     }
 
+    @ResponseStatus(HttpStatus.OK)
```
Comment (tbarabanov) [inline src/main/java/net/javaguides/springmvc/controller/rest/UserController.java]: use no content

File: src/main/java/net/javaguides/springmvc/exception/ResourceNotFoundException.java [hunk 5]
```diff
@@ -1,9 +1,28 @@
 package net.javaguides.springmvc.exception;
 
-public class ResourceNotFoundException extends Exception {
-    private static final long serialVersionUID = 1L;
+import org.springframework.http.HttpStatus;
+
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
```
Comment (tbarabanov) [inline src/main/java/net/javaguides/springmvc/exception/ResourceNotFoundException.java]: You do not need this annotation. Your errorHandler has one.

Comment (tburlakova) [reply src/main/java/net/javaguides/springmvc/exception/ResourceNotFoundException.java]: done

File: src/main/java/net/javaguides/springmvc/controller/rest/UserController.java (hunk 4 above)
Comment (tburlakova) [reply src/main/java/net/javaguides/springmvc/controller/rest/UserController.java]: done

**Pull request merged**